v23.1/cdc-queries.md

```diff
@@ -40,29 +38,32 @@ Parameter        | Description
+For a SQL diagram of the CDC query syntax, see the [`CREATE CHANGEFEED`](create-changefeed.html#synopsis) page.
+
+{% include_cached new-in.html version="v23.1" %} To emit different properties for a row, specify the following explicitly in CDC queries:
+
+- `cdc_prev`: A tuple-typed column that gives changefeeds access to the previous state of a row. See the [Emit the previous state of a row](#emit-the-previous-state-of-a-row) example for more detail.
+- CDC queries support [system columns](crdb-internal.html), for example:
+  - `crdb_internal_mvcc_timestamp`: Records the timestamp of each row created in a table. If you do not have a timestamp column in the target table, you can access `crdb_internal_mvcc_timestamp` in a changefeed. See the [Determine the age of a row](#determine-the-age-of-a-row) example.
 
 ## Limitations
 
-{% include {{ page.version.version }}/known-limitations/cdc-transformations.md %}
+{% include {{ page.version.version }}/known-limitations/cdc-queries.md %}
 
-## CDC transformation function support
+## CDC query function support
 
-You can use the following functions in CDC transformation queries:
+You can use the following functions in CDC queries:
 
 - Functions marked as "Immutable" on the [Functions and Operators page](functions-and-operators.html).
-- Changefeed functions:
-
-    {{site.data.alerts.callout_info}}
-    Changefeed functions are in preview and **subject to change** with continued development.
-    {{site.data.alerts.end}}
+- {% include_cached new-in.html version="v23.1" %} Changefeed functions:
 
     Function                  | Description
     --------------------------+----------------------
-    `cdc_is_delete() `        | Returns `true` if the event is a deletion event.
-    `cdc_prev()`              | Returns a JSON representation of a row's previous state. 
-    `cdc_updated_timestamp()` | Returns the event's update timestamp. This is typically the MVCC timestamp, but can differ, such as when the table is undergoing [schema changes](online-schema-changes.html).
+    `changefeed_creation_timestamp()` | Returns the changefeed creation timestamp.
+    `event_op()`              | Returns a string describing the type of event. If a changefeed is running with the [`diff`](create-changefeed.html#diff-opt) option, then this function returns `'insert'`, `'update'`, or `'delete'`. If a changefeed is running without the `diff` option, it is not possible to determine an update from an insert, so `event_op()` returns [`'upsert'`](https://www.cockroachlabs.com/blog/sql-upsert/) or `'delete'`.
+    `event_schema_timestamp()` | Returns the event's update timestamp. This is typically the MVCC timestamp, but can differ, such as when the table is undergoing [schema changes](online-schema-changes.html).
 
+- {% include_cached new-in.html version="v23.1" %} Non-volatile [user-defined functions](user-defined-functions.html).
```

Comment:
It would be nice to include an example for this somewhere;
```
CREATE FUNCTION doubleRevenue(r int)
                        -> RETURNS INT IMMUTABLE LEAKPROOF LANGUAGE SQL AS $$
                        ->   SELECT 2*r
                        -> $$;
CREATE CHANGEFEED ... AS SELECT doubleRevenue(rides.revenue::int) ...
```

Reply:
Added!